frb/galaxies/convert_tables.py

```diff
@@ -0,0 +1,39 @@
+""" Methods to convert data into Literature tables"""
+import os
+from pkg_resources import resource_filename
+
+import pandas
+
+from astropy.cosmology import Planck15
```

Comment:
Should be `Planck18`.  We should seriously consider using that alias.

Reply:
Maybe grep for `Planck15` in the repo just to be sure we're not missing any other place.